src/containers/PaddingSelector.tsx

```diff
@@ -58,29 +58,33 @@ const getDisplayedItems = memoize(10)(
       );
     }
 
-    if (list.length) {
```

Comment:
hey @rbardini, I removed this validation as it's not necessary given the outer validation of isEnabled which already takes into account the items.